doc/classes/Node.xml

```diff
@@ -977,7 +977,7 @@
 			Set the process thread group for this node (basically, whether it receives [constant NOTIFICATION_PROCESS], [constant NOTIFICATION_PHYSICS_PROCESS], [method _process] or [method _physics_process] (and the internal versions) on the main thread or in a sub-thread.
 			By default, the thread group is [constant PROCESS_THREAD_GROUP_INHERIT], which means that this node belongs to the same thread group as the parent node. The thread groups means that nodes in a specific thread group will process together, separate to other thread groups (depending on [member process_thread_group_order]). If the value is set is [constant PROCESS_THREAD_GROUP_SUB_THREAD], this thread group will occur on a sub thread (not the main thread), otherwise if set to [constant PROCESS_THREAD_GROUP_MAIN_THREAD] it will process on the main thread. If there is not a parent or grandparent node set to something other than inherit, the node will belong to the [i]default thread group[/i]. This default group will process on the main thread and its group order is 0.
 			During processing in a sub-thread, accessing most functions in nodes outside the thread group is forbidden (and it will result in an error in debug mode). Use [method Object.call_deferred], [method call_thread_safe], [method call_deferred_thread_group] and the likes in order to communicate from the thread groups to the main thread (or to other thread groups).
-			To better understand process thread groups, the idea is that any node set to any other value than [constant PROCESS_THREAD_GROUP_INHERIT] will include any children (and grandchildren) nodes set to inherit into its process thread group. this means that the processing of all the nodes in the group will happen together, at the same time as the node including them.
+			To better understand process thread groups, the idea is that any node set to any other value than [constant PROCESS_THREAD_GROUP_INHERIT] will include any child (and grandchild) nodes set to inherit into its process thread group. this means that the processing of all the nodes in the group will happen together, at the same time as the node including them.
```

Comment:
Do you mean calling them descendants? Also realized a punctuation error here will fix

Reply:
>  do you mean that we aren't using "descendant" or "ancestor" instead of "child/grandchild" or "parent/grandparent

I did mean this. A personal goal is

Ok, so.
There's the `is_ancestor_of` method, which implies that the word of a non-direct parent node is "ancestor", which means that "descendants" is the opposite.
And I would love to have this word consistent across the class reference, as it makes the whole concept easier to understand.
"child/grandchild" or "parent/grandparent" sounds good, great even, but it's about being consistent, and sadly the method isn't named `is_grandparent_of`.

Reply:
Sounds like a great future improvement

Reply:
Yeah, we need to discuss this another time. It's all over the place.

> It'd also involve, I'd say, cleaning up the use of "child(ren)" to mean not just direct children, as we somewhat commonly (or even regularly) use "child" to mean "descendant", which isn't unusual AFAIK in data structures

Once again same issue for consistency because `get_children` only returns immediate children, but making a distinction is way more important in Godot than your average data structure since some behavior only affects direct children (collision shapes, Controls as a whole, etc..)

Reply:
Right, but `find_children` is not immediate only, so we generally don't treat this as immediate only